tests/Unit/NumericalAlgorithms/Spectral/Test_Mesh.cpp

```diff
@@ -242,6 +243,29 @@ void test_serialization() noexcept {
               {{Spectral::Quadrature::GaussLobatto, Spectral::Quadrature::Gauss,
                 Spectral::Quadrature::GaussLobatto}}});
 }
+
+void test_option_parsing() {
+  INFO("Option Parsing creation");
+
+  std::array<std::string, 3> bases{"Chebyshev", "Legendre", "FiniteDifference"};
+  std::array<std::string, 2> quadratures{"Gauss", "GaussLobatto"};
```

Comment:
Make these a `const std::array<Spectral::Basis, 3>` and a `const std::array<Spectral::Quadratures, 2>`.

Reply:
And add `Quadrature::FaceCentered` and `Quadrature::CellCentered` please. We'll actually use those because commonly we want to interpolate to a uniform grid for visualization.